.claude/commands/04-testing/test.md

```
@@ -0,0 +1,122 @@
+---
+description: Comprehensive test generation and automation with coverage analysis
+allowed-tools: Bash(npm test:*), Bash(pytest:*), Bash(cargo test:*), Write, Edit, Read
+---
+
+# Comprehensive Testing
+
+## Test Environment Analysis
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)